packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/components/ButtonComponent.java

```diff
@@ -78,6 +76,9 @@ public Element render() {
         if (buttonStyle != null) {
             buttonElementPropsBuilder.style(buttonStyle);
         }
+        if (readOnly != null) {
```

Comment:
May be I'm wrong but I think `readOnly` cannot be `null`. If it is the case, could you move this to line 65 please?
And apply the same change to other widgets.

Reply:
`readOnly` can be null if the `isReadOnlyProvider` allow it, for the moment this is not the case with all the provider used, but can we guarantee that it'll be always the case.
With this check, we avoid any errors.